build.sbt

```diff
@@ -58,18 +58,21 @@ lazy val scalatraSettings = Seq(
       case Some((2, _)) =>
         unusedOptions ++ Seq(
           "-Xsource:3",
-          "-release:8",
+          "-release:" + {
+            val axes = virtualAxes.?.value.getOrElse(Nil)
+            (axes.contains(javax), axes.contains(jakarta)) match {
+              case (true, false) => "8"
+              case (false, true) => "17"
+              case _ => "8"
+            }
+          },
           "-Xlint",
           "-Xcheckinit",
         )
       case _ =>
         Nil
     }
   },
-  javacOptions ++= Seq(
```

Comment:
I got rid of this section because, from what I understand, this section is only used to compile `.java` files, isn't it?